### DISCUSSION TITLE
Group ID is not found - after several days

### DISCUSSION THREAD

**Discussion by ulilamri419** [kirimwa-id/forum#27] [Tanya Jawab]
Dear Mas @rioastamal 

Ada isu / bug kah, terkait error message "Group ID is not Found" ketika sending message ke group. Seperti tampilan berikut:
![image](https://user-images.githubusercontent.com/70904189/130543948-3bf4003b-f915-4d62-b261-69efa8c4fdbd.png)

Setelah saya sync ulang "Get group ID" saya tes kembali baru bisa dan sukses.

Mohon di cek apakah, ada limitasi sync ini misal harus per 1x24 or bagaimana? 
Karena saya test terakhir (tgl 16-Agustus) dan message auto generate H+4 (96 jam) (tgl 20) error tidak bisa kirim (spt saya infokan di awal tadi).

Terima Kasih.
Amri

**Comment by rioastamal** [comment]
Halo mas @ulilamri419,

Terima kasih feedbacknya. Daftar contact diambil langsung dari device tanpa process caching. Sehingga jika device disconnect maka data di memory akan hilang atau ketika process diserver restart. 

Sepertinya perlu ditambahkan cache, sehingga ketika device terputus dan terjadi koneksi ulang, maka contact dalam status masih di-sync. Saat masih dalam sync itulah cache akan berguna sehingga kemungkinan Group ID not found tidak terjadi ketika contact sudah ter-sync sebelumnya.

Sudah saya masukkan pada roadmap #28

**Comment by rioastamal** [comment]
Halo Mas @ulilamri419, fitur sudah di-implementasikan di v1.0-RC5 #20. Harusnya sekarang meskipun dari kondisi disconnected error `Group ID not found` tidak terjadi.
